src/components/Profile/Profile.js

```diff
@@ -6,154 +6,179 @@ import { Star } from '../styleElements/icons';
 import { LogoActive } from '../styleElements/icons';
 import { ChevronRight } from '../styleElements/icons';
 import { dummyData } from '../MatchPage/MatchCard/dummyData';
+import { useDispatch } from 'react-redux';
```

Comment:
I added these two imports (I point this out because most of the changes here are just because my auto-formatting is different so it's hard to tell)